sql/14-increase_owner_pubkey.sql

```diff
@@ -0,0 +1,12 @@
+--
+-- Increase the char limit of owner_public_key from 256 to 512.
+--
+
+-- pew-pew
+\connect blobber_meta;
+
+-- in a transaction
+BEGIN;
+    ALTER TABLE allocations
+        ALTER COLUMN owner_public_key varchar(512) NOT NULL;
```

Comment:
```
postgres-post_1  | 
postgres-post_1  | /blobber/bin/postgres-entrypoint.sh: running /blobber/sql/14-increase_owner_pubkey.sql
postgres-post_1  | You are now connected to database "blobber_meta" as user "postgres".
postgres-post_1  | BEGIN
postgres-post_1  | psql:/blobber/sql/14-increase_owner_pubkey.sql:11: ERROR:  syntax error at or near "varchar"
postgres-post_1  | LINE 2:         ALTER COLUMN owner_public_key varchar(512) NOT NULL;
postgres-post_1  |                                               ^
postgres-post_1  | ROLLBACK
postgres-post_1  | 
postgres-post_1  | 
blobber1_postgres-post_1 exited with code 0

```

Reply:
OK I pushed a fix that should fix it for postgres 9.1 and above